src/resources/action-map.js

```diff
@@ -237,6 +239,22 @@ return scale * value
         }
       }
     }),
+    '==': (l, r, scaleField = 1) => {
```

Comment:
```suggestion
    '==': (l, r, scaleValue = 1) => {
```

Maybe better? Why do you call it field?

Reply:
Because of below: https://github.com/loopmediagroup/es-alchemy/pull/235/files#diff-25f46d6e6d985a6773bdb10577e2ed52R252